### DISCUSSION TITLE
Problematic surface data in NorESM2.5 SE simulations

### DISCUSSION THREAD

**Discussion by kjetilaas** [NorESMhub/noresm3_dev_simulations#38] [CLM-FATES grid and resolution]
We recently discovered that the surface datasets for CLM used in ne30 simulations have some problematic values. See details in this thread: 
[https://github.com/ESCOMP/CTSM/issues/2744](https://github.com/ESCOMP/CTSM/issues/2744)

This bug has been there in the surface dataset for all NorESM2.5 simulations using ne30 (SE) grid. It is currently unclear how strong effect this has on the simulations, but the affected variables (mainly soil texture variables) are important for the calculation of dust emissions. It is therefore possible that this is contributing to the high dust biases seen in initial NorESM2.5 test. However, it is mainly Africa and some parts of Asia that are highly affected, while most of Australia seems unaffected by this, which is where the larges dust biases where seen. 

@oyvindseland @adagj @DirkOlivie

**Comment by maritsandstad** [comment]
There now seems to be updated surface datasets. @mvertens do you know what we might want to redo of simulation with new datasets, and do we have updated tags with the correct dataset?

**Reply by kjetilaas** [reply]
Thanks for mentioning this, Marit! I agree, we should do some test to see how big the effect of this is. 
At this point, we probably don't want to redo all the simulations with NorESM2.5 since these are anyway uncalibrated initial tests. But it would be good to redo one where this is the only change.

I know they are making new IC files at NCAR. We could either wait for this, or redo one of the simulations we have/had planned with FATES, where we at this point have only done short (50-100 yr), offline spin-ups (in SP and NOCOMP mode). So this would not be a big effort to redo. 

Any thoughts on this, @mvdebolskiy?

**Reply by mvdebolskiy** [reply]
The spinup has to be redone later anyway. For now we just need to check what is affected (apart from dust) by the surface dataset problem/

**Comment by MichaelSchulzMETNO** [comment]
Well - the idea of a rerun of one of the 2.5 experiments is just to see, if indeed the dust emissions are affected by the "spectral element" surface fields in CTSM. 1 year with corrected surface properties might be enough to see if dust emissions indeed change.
